nl/references/tested-judge/config/README.md

```diff
@@ -0,0 +1,363 @@
+
+De linters worden ingesteld in het object bijhorend bij het veld `linter`.
+De sleutels van dit object zijn de ondersteunde programmeertalen.
+Met een boolean wordt aangegeven of de linters gebruikt moeten worden.
+Standaard zullen de linters voor elke programmeertaal gebruikt worden.
+
+Voorbeeld uitschakelen linters:
+```json
+{
+  "evaluation": {
+    "options": {
+      "linter": {
+        "c": false,
+        "haskell": false,
+        "java": false,
+        "javascript": false,
+        "kotlin": false,
+        "python": false
+      }
+    }
+  }
+}
+```
+
+## Programmeertaal specifiek
+Naast de configuratieopties voor TESTed zelf, bestaan er ook per programmeertaal specifieke optionele opties.
+Deze opties worden ingesteld in het object bijhorend bij het veld `evaluation.language`.
+
+### C
+De programmeertaal C heeft geen specifiek opties.
+
+### Haskell
+De programmeertaal Haskell heeft 1 optie: `hlint_config`.
+Deze verwacht de bestandsnaam van een HLint configuratiebestand.
+Dit bestand moet zich in de `evaluation` map van de Dodona oefening bevinden.
+
+Voorbeeld HLint configuratie:
+```json
+{
+  "evaluation": {
+    "options": {
+      "language": {
+        "haskell": {
+          "hlint_config": "hlint.config.yaml"
+        }
+      }
+    }
+  }
+}
+```
+
+### Java
+De programmeertaal Java heeft 1 optie: `checkstyle_config`.
+Deze verwacht de bestandsnaam van een Checkstyle configuratiebestand.
+Dit bestand moet zich in de `evaluation` map van de Dodona oefening bevinden.
+
+Voorbeeld Checkstyle configuratie:
+```json
+{
+  "evaluation": {
+    "options": {
+      "language": {
+        "java": {
+          "checkstyle_config": "java_style.xml"
+        }
+      }
+    }
+  }
+}
+```
+
+### JavaScript
+De scripttaal JavaScript heeft 1 optie: `eslint_config`.
+Deze verwacht de bestandsnaam van een ESLint configuratiebestand.
+Dit bestand moet zich in de `evaluation` map van de Dodona oefening bevinden.
+
+Voorbeeld ESLint configuratie:
+```json
+{
+  "evaluation": {
+    "options": {
+      "language": {
+        "javascript": {
+          "eslint_config": "eslintrc.yaml"
+        }
+      }
+    }   
+  }
+}
+```
+
+### Kotlin
+De programmeertaal Kotlin heeft 4 opties:
+`editorconfig`, `disabled_rules_ktlint`, `ktlint_ruleset` en `ktlint_experimental`.
+Al deze opties worden gebruikt door de linter *KTLint*.
+- `editorconfig`: bestandsnaam van een `.editorconfig` bestand (zie <https://editorconfig.org/>) in de `evaluation` map
+  van de Dodona oefening.
+- `disabled_rules_ktlint`: een lijst van *KTLint* regels die genegeerd mogen worden.
+  Er kan ook gebruik gemaakt worden van een kommagescheiden string van regels.
+- `ktlint_ruleset`: Een bestandsnaam van een JAR-bestand met extra regels.
+  Dit bestand moet zich in de `evaluation` map van de Dodona oefening bevinden.
+- `ktlint_experimental`: Duid aan of de experimentele linter regels gebruikt mogen worden.
+  Standaard zullen deze regels gebruikt worden.
```

Comment:
Is er een reden om experimentele dingen aan te zetten? Ik veronderstel dat ze experimenteel zijn omdat ze misschien nog _false positives_ geven, maar dat willen we absoluut vermijden bij oefeningen.

Reply:
I have enabled them by default, because we first want to have all linter rules enabled before limiting the linter rules used by TESTed for the student feedback.